index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta http-equiv="X-UA-Compatible" content="ie=edge">
+        
+        <link href="https://fonts.googleapis.com/css?family=Josefin+Sans:300,400,400i|Nunito:300,300i" rel="stylesheet">
+        <link rel="stylesheet" href="css/style.css">
+        <link rel="shortcut icon" type="image/png" href="img/favicon.png">
+        
+        <title></title>
+    </head>
+
+    <body>
+        <main class="Main-container">
+            <header class="Header">
+                <nav class="container">
+                    <div class="logoplaceholder">
+                        <a href="#" class="logo__link"><h1 class="logo">Welcome</h1></a>
+                        
+                    </div>
+                    <div class="menu__button">
+                        <div class="Menu__Icon">
+                            <a href="#" class="menu__link">
+                                <img src="img/Icon - Menu.svg" alt="Menu Icon">
+                            </a>
+                        </div>
+                    </div>
+    
+                </nav>
+            </header>
+            <section class="Headline">
+                <div class="Headline__primary">
+                   <h2 class="Text__intro"> <em>Hey there. I’m Solomon </em></em></h2>  <br> <h2 class="Name_title"> <em>I’m a software developer</h2></em>
```

Comment:
As mentioned by a previous reviewer, in the Figma design, this text should not be italic. Please pay attention to detail and remove the `<em>` from the text.

![Screenshot from 2022-09-23 09-45-30](https://user-images.githubusercontent.com/16392046/191908103-ae269679-609c-4ba9-bb9d-f63de94cc823.png)

[Optional]
Also since this is the most important topic on the page please consider wrapping it in an `h1` tag instead of an `h2` tag.